src/app/payment/store/checkoutReducer.ts

```diff
@@ -6,6 +6,12 @@ export const initialStateForCheckout: State = {
   promoCodeName: '',
   avatarBlob: null,
   isPaying: false,
+  isCheckoutReadyToRender: false,
+  isUpdateSubscriptionDialogOpen: false,
+  isUpdatingSubscription: false,
+  isUpsellSwitchActivated: false,
+  country: '',
+  prices: [],
```

Comment:
why can it have a price list instead of a price?

Reply:
Okay, we'll discuss whether it's better to pass only one price, I don't have enough context right now. 
And if we finally have to change it, we'll change it in another task.